sklearn/cross_decomposition/_pls.py

```diff
@@ -780,6 +780,9 @@ def __init__(
 class CCA(_PLS):
     """Canonical Correlation Analysis, also known as "Mode B" PLS.
 
+    For an example on usage, see
```

Comment:
You could have a better description like:

For a comparison with other algorithms such as .....